src/pages/project/ProjectCreate.js

```diff
@@ -36,12 +36,12 @@ function ProjectCreate() {
       Object.entries(formDataValue).forEach(([key, value]) => {
         formData.append(key, value)
       })
-      const result = await dispatch(postProject(formData))
-      if (result.payload.status === 200) {
-        navigate(`/project/${result.payload.data._id}`)
+      const result = await addProject(formData).unwrap()
+      if (result) {
```

Comment:
I think we need to check here if result is actually successful result. 
Now it will redirect in case of error as well.